lib/loaders/KCircularLoader.vue

```diff
@@ -105,6 +113,13 @@
         type: Boolean,
         default: false,
       },
+      /**
+       * Do not hide the loader until `freeze` in milliseconds. Useful to avoid jarring UX when the actions finishes very fast. In comparison to `delay`, `freeze` emhpasises that an action associated with the loader is indeed taking place. `show` needs to be used instead of `v-if/v-show` for this to work.
```

Comment:
this comment explaining the difference from delay 

> emhpasises that an action associated with the loader is indeed taking place

is excellent!

Very small spelling nit-pick: emhpasises should be emphasizes